fuse_core/test/test_transaction.cpp

```diff
@@ -51,16 +52,16 @@ using fuse_core::UUID;
  *
  * Order of the stamps is not important. Extra stamps in the Transaction will return False.
  *
- * @param expected_stamps  The set of expected stamps
- * @param transaction      The transaction to test
- * @return                 True if the expected stamps, and only the expected stamps, exist in the
- *                         transaction, False otherwise.
+ * @param expected     The set of expected stamps
+ * @param transaction  The transaction to test
+ * @return             True if the expected stamps, and only the expected stamps, exist in the
+ *                     transaction, False otherwise.
  */
-bool testInvolvedStamps(
-  const std::vector<ros::Time>& expected_stamps, const Transaction& transaction)
+template <typename TimeRange>
```

Comment:
You can also add `@tparam TimeRange` to add a doxygen comment for this template param: http://www.doxygen.nl/manual/commands.html#cmdtparam